posthog/event_usage.py

```diff
@@ -196,6 +196,27 @@ def report_bulk_invited(
     )
 
 
+def report_user_organization_membership_level_changed(
+    user: User,
+    organization: Organization,
+    new_level: int,
+    previous_level: int,
+) -> None:
+    """
+    Triggered after a user's membership level in an organization is changed.
+    """
+    posthoganalytics.capture(
+        user.distinct_id,
+        "membership level changed",
+        properties={
+            "organization_id": str(organization.id),
```

Comment:
Isn't this redundant with the `organization` _group_ already associated with the event?